docs/modules/ROOT/pages/python-client-getting-started.adoc

```diff
@@ -0,0 +1,195 @@
+= Getting Started with the Hazelcast Python Client
+:page-layout: tutorial
+:page-product: imdg
+:page-categories: Caching, Getting Started
+:page-lang: python3
+:page-est-time: 5-10 mins
+:description: This tutorial will get you started with the Hazelcast Python client.
+
+== What You'll Learn
+
+{description}
+
+== Before you Begin
+
+* A text editor or IDE
+* Docker
+* Python 3.6+
+
+== Start a Hazelcast Member
+
+We will use the 5.3 version of Hazelcast for this tutorial.
+
+In this tutorial, we will use https://hazelcast.com/products/viridian/[Hazelcast Viridian Cloud].
+
+1. Sign up for a Hazelcast Viridian Cloud account (Free accounts are available).
+2. Log in to your Hazelcast Viridian Cloud account and navigate to your https://viridian.hazelcast.com[cluster list].
+3. Create a new cluster or start an existing one.
+
+
+== Install Hazelcast Python Client
+
+1. Identify the Hazelcast Viridian Cloud https://viridian.hazelcast.com[cluster] you want to connect to and choose Connect Cluster.
+2. Use the quick connection guide to download the preconfigured Python sample client.
+3. Follow the instructions on the quick connection guide to start using your cluster.
+
+To understand and use the client, review the https://hazelcast.readthedocs.io/en/stable/client.html#hazelcast.client.HazelcastClient[Python API documentation] to better understand what is possible.
+
+== Starting the Python Client
+
+The following section creates and starts a new Hazelcast client with default configuration.
+
+[source,python]
+----
+import hazelcast
+
+client = hazelcast.HazelcastClient()
+
+client.shutdown()
+----
+
+== Use Map
+
+A Hazelcast map is a distributed key-value store, similar to Python dictionary. You can store key-value pairs in a map.
+In the following example, we will work with map entries where the keys are ids and the values are defined as an object representing a city.
```

Comment:
the example uses string for the value. should we call it an object or string?

Reply:
All our guides have the version specified - this is a consistency copy from C# and Java.

Suggest fixing them ALL after the first cut is made available.

Reply:
OK.